src/daemons/rhsmcertd-worker.py

```diff
@@ -42,10 +43,19 @@
 _ = gettext.gettext
 
 
+def exit_on_signal(*args):
+    sys.exit(0)
```

Comment:
@awood, because a signal handler takes two args: the signal number and the stack frame (reference: https://docs.python.org/2/library/signal.html#signal.signal). We don't really care about either of those here. I suppose we could rename the parameter to `_` (i.e. `exit_on_signal(*_)`) though to better communicate we don't care about args. Thoughts?